src/credential/sd_jwt_vc.rs

```diff
@@ -0,0 +1,169 @@
+use base64::prelude::*;
+use ssi::{
+    claims::{
+        jwt::{AnyClaims, JWTClaims},
+        sd_jwt::{RevealedSdJwt, SdJwtBuf},
+        vc::{v1::Credential as _, v2::Credential as _},
+        JwsString,
+    },
+    prelude::AnyJsonCredential,
+};
+use uuid::Uuid;
+
+use crate::{CredentialType, KeyAlias};
+
+use super::VcdmVersion;
+
+#[derive(uniffi::Object, Debug, Clone)]
+/// A verifiable credential secured as a JWT.
+pub struct SdJwtVc {
```

Comment:
What is the purpose of this type exactly? Where do we need it? I don't know much about `uniffi` but I don't see any constructor for it so I don't think we can even build it.

Reply:
We might need a type like this similar to the types in the `credentials/` module for presentation. Will open a new PR to address this, as needed.